scylladb/sphinx-scylladb-theme
# Modernize packaging of static files

Closes https://github.com/scylladb/sphinx-scylladb-theme/issues/3

This PR: 
- Introduces webpack bundling and minifying static assets.
- Removes unused images and scripts.
- Removes saved Foundation, jQuery, and Modernizr and use the NPM versions.
- Adds linters to prettify CSS and JS.

Having the JS and part of the CSS organized at the beginning should speed up the implementation of the new theme.
Instead of merging these changes directly in the master branch, I propose to use the branch ``new-design`` to test exhaustively in a staging environment all the changes related to the redesign project before integrating them in the master branch.

## How to test this PR

1. ``npm run build`` should run without errors.
2. ``cd docs`` & ``make preview`` should render the site without errors.
